ivy/functional/frontends/tensorflow/raw_ops.py

```diff
@@ -261,18 +256,10 @@ def Square(*, x, name="Square"):
 Sub = tf_frontend.subtract
 
 
-
 def Sum(*, input, axis, keep_dims=False, name="Sum"):
     return ivy.astype(ivy.sum(input, axis=axis, keepdims=keep_dims), input.dtype)
 
 
-def Tan(*, x, name="Tan"):
```

Comment:
Why modify the contents of` Tf raw_ops.py file`? Could you kindly revert these changes if they were made by accident.